tasks/createReleasePR.js

```diff
@@ -60,15 +60,16 @@ if (require.main === module) {
   cli({}, async () => {
     console.log(`Making release...`)
     const changeLog = fs.readFileSync(__dirname + `/../CHANGELOG.md`, `utf8`)
+    const pending = fs.readFileSync(__dirname + `/../PENDING.md`, `utf8`)
```

Comment:
just nice to have for compatibility:
- we should create paths using the OS agnostic path.join(__dirname, '..', 'PENDING.md')